app/api/server/lib/email-channel.js

```diff
@@ -0,0 +1,63 @@
+import { EmailChannel as EmailChannelRaw } from '../../../models/server/raw';
```

Comment:
Please, rename this file to `emailChannel.js`.
Also, why didn't you implement this as `ts` file?